src/errors/customApi.js

```diff
@@ -0,0 +1,7 @@
+class CustomAPIError extends Error {
```

Comment:
right now this CustomAPIError is not used anywhere. Is it for future purposes or am I missing something?